google-cloud-bigtable/pom.xml

```diff
@@ -338,6 +338,44 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>bigtable-directpath-ipv4-it</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>directpath-it</id>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <skip>false</skip>
+
+                  <systemPropertyVariables>
+                    <bigtable.env>cloud</bigtable.env>
+                    <bigtable.data-endpoint>${bigtable.directpath-data-endpoint}</bigtable.data-endpoint>
+                    <bigtable.admin-endpoint>${bigtable.directpath-admin-endpoint}</bigtable.admin-endpoint>
+                    <bigtable.grpc-log-dir>${project.build.directory}/test-grpc-logs/directpath-it</bigtable.grpc-log-dir>
+                    <bigtable.attempt-directpath>true</bigtable.attempt-directpath>
+                    <bigtable.directpath-ipv4>true</bigtable.directpath-ipv4>
+                  </systemPropertyVariables>
+                  <includes>
+                    <!-- TODO(igorbernstein): Once the control plane is accessible via directpath, add admin tests -->
+                    <include>com.google.cloud.bigtable.data.v2.it.*IT</include>
+                  </includes>
+                  <summaryFile>${project.build.directory}/failsafe-reports/failsafe-summary-directpath-it.xml</summaryFile>
```

Comment:
sorry, just noticed this, I think this has been its own summary file so that it wont clobber the regular directpath-it tests

Reply:
Good catch. Also fixed grpc-log-dir.